src/components/Layout/Layout.module.css

```diff
@@ -121,7 +121,7 @@
  */
 .layout--sidebar-wide > .layout__section--sidebar {
   @media all and (min-width: $eds-bp-xl) {
-    width: 22rem; /* TODO: Make more dynamic */
```

Comment:
There's no specific thing to change here except that there may be a better way to define width. created a ticket for the layout components to revisit them